huggingface_sb3/load_from_hub.py

```diff
@@ -17,19 +17,19 @@ def load_from_hub(repo_id: str, filename: str) -> str:
         )
 
     try:
-      # Get the model from the Hugging Face Hub
-      model_download_url = hf_hub_url(repo_id=repo_id,
-                                 filename=filename)
-      # Downloading and caching the model on your local disk
-      downloaded_model_file = cached_download(model_download_url,
-                                             library_name="huggingface-sb3",
-                                             library_version="1.0.7")
-    
+        # Get the model from the Hugging Face Hub
+        model_download_url = hf_hub_url(repo_id=repo_id,
+                                        filename=filename)
+        # Downloading and caching the model on your local disk
+        downloaded_model_file = cached_download(model_download_url,
+                                                library_name="huggingface-sb3",
+                                                library_version="1.0.7")
+
     # TODO: I need to find a way to override the HTTPError with mine
     except HTTPError:
-      raise HTTPError(
-          f"This {filename} does not exist. Please verify \n"
-          f"1. That your repository exists https://huggingface.co/{repo_id} \n"
-          f"2. That the filename you want to retrieve is the correct one and is a .zip file"
-      )
+        raise HTTPError(
```

Comment:
Is this really needed? What error would `hf_hub_download` give the user if it points to a wrong repo or a wrong file?

Reply:
For now this part does nothing because I didn't overwrite the http error message of hf_hub_download.